src/components/Table/readme.md

```diff
@@ -695,3 +695,216 @@ const tableContainerStyles = { height: 300 };
+    if(value === 'verified'){
+        return <Badge label={value} variant="outline-brand"/>;
+    }
+    return <Badge label={value} variant="lightest" />;
+};
+
+const data = [
+    {
+        name: 'Leandro Torres',
+        company: 'nexxtway',
+        email: 'leandro@gmail.com',
+        status: 'verified',
+        id: '1234qwerty',
+        link: '#',
+    },
+    {
+        name: 'Jose Torres',
+        company: 'Google',
+        email: 'jose@gmail.com',
+        status: 'pending',
+        id: '1234asdfgh',
+        link: '#',
+    },
+    {
+        name: 'Reinier',
+        company: 'nexxtway',
+        email: 'reinier@gmail.com',
+        status: 'verified',
+        id: '1234zxcvbn',
+        link: '#',
+    },
+    {
+        name: 'Sara',
+        company: 'nexxtway',
+        email: 'sara@gmail.com',
+        status: 'verified',
+        id: '5678qwerty',
+        link: '#',
+    },
+    {
+        name: 'Tahimi',
+        company: 'nexxtway',
+        email: 'tahimi@gmail.com',
+        status: 'verified',
+        id: '5678asdfgh',
+        link: '#',
+    },
+    {
+        name: 'Leo',
+        company: 'nexxtway',
+        email: 'leo@gmail.com',
+        status: 'verified',
+        id: '5678zxcvbn',
+        link: '#',
+    },
+    {
+        name: 'Tahimi Leon',
+        company: 'nexxtway',
+        email: 'leon@nexxtway.com',
+        status: 'verified',
+        id: '9012qwerty',
+        link: '#',
+    },
+    {
+        name: 'Alejandro',
+        company: 'Google',
+        email: 'alejandro@gmail.com',
+        status: 'verified',
+        id: '5678zdfgdf',
+        link: '#',
+    },
+    {
+        name: 'Carlos',
+        company: 'Oracle',
+        email: 'carlos@gmail.com',
+        status: 'verified',
+        id: '3434fgfgdf',
+        link: '#',
+    },
+    {
+        name: 'Luis',
+        company: 'Google',
+        email: 'luis@gmail.com',
+        status: 'verified',
+        id: '9012asdfgh',
+        link: '#',
+    },
+];
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+`;
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+`;
+
+const StyledLink = styled.a.attrs(props => props.theme.rainbow)`
+    color: ${props => props.palette.brand.main}
+    &:hover {
+        color: ${props => props.palette.brand.main};
+    }
+    &:visited {
+        color: ${props => props.palette.brand.main};
+    }
+    &:active {
+        color: ${props => props.palette.brand.main};
+    }
+`;
+
+const ProfileLink = ({value, row}) => <StyledLink href={row.link}>{ value }</StyledLink>;
+
+function TableExample (props) {
+    const [ data, setData ] = useState(props.data);
+    const [ sortedBy, setSortedBy ] = useState(undefined);
```

Comment:
you don't need to initialize state explicit to undefined since when using ```useState()``` it is already undefined